www-servers/lighttpd/metadata.xml

```diff
@@ -10,12 +10,12 @@
 		<name>Proxy Maintainers</name>
 	</maintainer>
 	<longdescription lang="en">
-		lighttpd is a secure, fast, compliant and very flexible web-server
-		which has been optimized for high-performance environments. It has a very
-		low memory footprint compared to other webservers and takes care of cpu-load.
-		Its advanced feature-set (FastCGI, CGI, Auth, Output-Compression,
-		URL-Rewriting and many more) make lighttpd the perfect webserver-software
-		for every server that is suffering load problems.
+		lighttpd (pronounced /lighty/) is a secure, fast, compliant, and very
+		flexible web server that has been optimized for high-performance
+		environments. lighttpd uses memory and CPU efficiently and has lower
+		resource use than other popular web servers. Its advanced feature-set
+		(FastCGI, CGI, Auth, Output-Compression, URL-Rewriting and much more)
+		make lighttpd the perfect web server for all systems, small and large.
 	</longdescription>
 	<use>
 		<flag name="brotli">Enable output compression via Brotli (recommended)</flag>
```

Comment:
Nit:
```suggestion
		<flag name="brotli">Enable output compression via <pkg>app-arch/brotli</pkg> (recommended)</flag>
```

I wonder if we should do (for this and others) e.g. "Brotli (<pkg>app-arch/brotli</pkg>)" instead? Or maybe just do it where it's less obvious? e.g. maybe for libmaxminddb vs MaxMindDB?

Reply:
I defer to your recommendations here.  I think linking is good.